networks/ethereum/tokens/0x6de037ef9ad2725eb40118bb1702ebb27e4aeb24.json

```
@@ -0,0 +1,14 @@
+{
+  "CMCslug": "render",
+  "decimals": 18,
+  "mldId": "ethereum-RENDER-0x6de037ef9ad2725eb40118bb1702ebb27e4aeb24",
+  "orgName": "Render Network",
+  "orgUrl": "https://rendernetwork.com/",
+  "tokenAddress": "0x6de037ef9ad2725eb40118bb1702ebb27e4aeb24",
+  "tokenColorTheme": 20,
+  "tokenIcon": "/static/tokens/RENDER.svg",
+  "tokenName": "Render Token",
+  "tokenTicker": "RENDER",
```

```suggestion
  "tokenTicker": "RNDR",
```
According to the token symbol of the token: https://etherscan.io/token/0x6de037ef9ad2725eb40118bb1702ebb27e4aeb24